paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -38,5 +50,56 @@ def main():
     )
 
 
+def run_local_zookeeper(user, zookeeper_port):
+    client = get_docker_client()
+    containers = client.containers()
+    zookeeper_container = next(
+        (
+            container
+            for container in containers
+            if container["Names"] == [f"/{user}-paasta-zookeeper"]
+        ),
+        None,
+    )
+    if zookeeper_container is None:
+        create_zookeeper_container(zookeeper_port, user)
+    elif zookeeper_container.get("Status") != "running":
+        client.remove_container(zookeeper_container, force=True)
+        create_zookeeper_container(zookeeper_port, user)
+
+
+def create_zookeeper_container(port, user):
```

Comment:
```suggestion
def create_zookeeper_container(port: int, user: str) -> None:
```

Reply:
additionally, perhaps this should be `run_zookeeper_container`?

Reply:
finally (and feel free to drop since i don't think anyone else would care): would be nice for consistency if the parameter ordering matched the `run_local_zookeeper` above (it's the same params in a different order :p)